css/styles.css

```diff
@@ -0,0 +1,417 @@
+html{
+  overflow-x: hidden;
+}
+.navbar figure{
+  width: 300px;
+  height: 130px;
+}
+.navbar figure img{
+  height: 100%;
+  width: 100%;
+}
+.font-lg-2{
+  font-size: 1.35em;
+}
+.bg-red{
+  background-color: red;
+}
+.cl-red{
+  color: red;
+}
+.nav-1{
+  height: 100px;
+  z-index: 3;
+}
+.container-1{
+  width: 96%;
+  margin: 0 auto;
+}
+.navbar-header{
+  float: left;
+  padding: 15px;
+  text-align: center;
+  width: 100%;
+}
+.navbar-brand{
+  float: none;
+}
+.nav-1 .navbar-brand svg{
+  display: block;
+  margin-bottom: -4vh;
+  width: 350px;
+  height: 65px;
+  margin-right: 15px;
+  fill: #fff;
+}
+.nav-sticky .navbar-brand svg{
+  text-align: left;
+  display: block;
+  width: 120px;
+  margin-right: 15px;
+  fill: #fff;
+}
+.navbar-header{
+  padding: 8px;
+}
+footer svg{
+  fill: red;
+  display: block;
+  width: 200px;
+  height: 40px;
+}
+.navbar .w-25{
+  margin-bottom: -6vh;
+}
+.navbar .fl-right a:nth-child(2){
+  background-color: #fff;
+  padding: 8px;
+  margin-left: 10px;
+}
+.nav-2::before{
+  position: absolute;
+  content: '';
+  top: 100px;
+  right: 0;
+  width: 66px;
+  display: block;
+  background-color: #f72210;
+  border-left: 13px solid #ffff;
+  border-top: 12px solid #f72210;
```

Comment:
`border-top: 12px solid #f72210;` is unnecessary the top border color is set by `background-color: #f72210;` on line 77.

Reply:
The border top that has the same color with the background-color is not by mistake. This is in order to achieve the desired effect on the on the nav-2:before.